.github/workflows/style.yml

```diff
@@ -27,6 +27,7 @@ jobs:
 
     - name: Install dependencies
       run: |
+        pip install .
```

Comment:
Do we need to install komodo too for style checking?

Reply:
yes, there was a reason, need to rething what it was..

Reply:
It is in order to obtain the dependencies